### ISSUE TITLE
[Accessibility]: Event is bubbled-up when clickable content is projected in clr-dg-column

### ISSUE THREAD

**Issue by Tushar2098** [vmware/clarity#4422]
## Describe the bug

Sorting is executed when user tries to open the signpost (though keyboard) which is passed as projected content to `clr-dg-column`

Even with `event.stopPropogation` column button's event is getting triggered.

**Column with Signpost**

```
<clr-dg-column [clrDgField]="'name'">
  Name
  <clr-signpost class="signpost">
    <a href="javascript:void(0)" (click)="anchor($event)" style="padding-left:10px" clrSignpostTrigger>
      <clr-icon shape="info-circle" size="17"></clr-icon>
    </a>
    <clr-signpost-content *clrIfOpen>
      <h3>Default Signpost</h3>
      <p>Position: <code class="clr-code">right-middle</code></p>
    </clr-signpost-content>
  </clr-signpost>
</clr-dg-column>

```

**Event Handler**
```
  anchor(event) {
    event.preventDefault();
    event.stopPropagation();
    console.log('Anchor');
  }
```

## How to reproduce

1.  Go to [Stackblitz](https://stackblitz.com/edit/a11y-grid?file=src/app/app.component.html)
2.  Tab on the info icon
3.  Notice signpost content is not opened but instead sorting is executed

## Expected behavior

Signpost content should be opened and sorting should not happen

## User impact
Vision-impaired users and keyboard users cannot get the info from signpost.

## Versions

**App**

* Angular: 8.2.14
* Clarity: 2.1.1

**Device:**

* OS: Windows
* Browser: Chrome/Firefox
* Version 80.0.3987.149 / 74.0

**Comment by hippee-lee** [comment]
Hi @Tushar2098 - this happens because both the column label (Name) and the signpost are projected content. When there is sorting on a column the projected content is put inside the button that toggles sorting. 

For that reason I would not recomend putting a signpost inside a column that needs to be sortable. Even if the signpost was able to be moved outside the button it wuould men four small clickable elements in the column header [text label, sorrting arrow icon, signpost & filter icon). I know one of the reasons we didn't add a clear sort icon is becuase of the extra clickable element inside a small space. 

I'm going to close this as something we wont fix but if there is somehting I'm missing or haven't thought of please continue the discussion here.

**Comment by Tushar2098** [comment]
@hippee-lee Thanks for the quick response.

Your point is correct. But the behavior should be consistent across all the platforms. 
**I am not seeing this issue in macOS** I don't understand how `event.stopPropagation();` is working in MacOS and not in Windows. Please let me know if I am missing something. 

Thanks!

**Comment by hippee-lee** [comment]
Hi @Tushar2098 - can you provide a stackblitz (https://stackblitz.com/@clr-team) that is a simnple reproduction of the implementation you are seeing htis in?

If you can provide me with the simple reproduction, I'll look at it on both macOS and Windows to see if I can provide any more info here.

**Comment by Tushar2098** [comment]
@hippee-lee  

FYI...
[https://stackblitz.com/edit/a11y-grid](https://stackblitz.com/edit/a11y-grid) 

Thanks!

**Comment by hippee-lee** [comment]
@Tushar2098 - this demo is still projecting the link inside of the button. 
As mentioned before this is an issue due to the content (link for signpost) projected into a button. The fix here is to either remove sorting from the column or remove the signpost and use a different design to communicate that information.
